docs/Brewfile

```diff
@@ -0,0 +1,8 @@
+brew "git"
+brew "helm"
+brew "helmfile"
+brew "jq"
+brew "vault"
+cask "docker"
+cask "google-cloud-sdk"
+cask "intellij-idea"
```

Comment:
I'm trying this out to see what happens. So far it failed to install docker because I already had docker desktop installed. Probably the same thing will occur with google cloud and IntelliJ.
```
~/source/jade-data-repo/docs # brew bundle install
Updating Homebrew...
Using git
Using helm
Using helmfile
Using jq
Using vault
Installing docker
==> Downloading https://desktop.docker.com/mac/main/amd64/69386/Docker.dmg
==> Installing Cask docker
Error: It seems there is already an App at '/Applications/Docker.app'.
==> Purging files for version 4.1.0,69386 of Cask docker
Installing docker has failed!
Installing google-cloud-sdk
```